src/core/body/body.view.d.ts

```diff
@@ -0,0 +1,13 @@
+import {View} from '../view/view';
+
+
+export declare class BodyView extends View {
+	constructor(model: object, table: object);
+
+	table: object;
```

Comment:
remove table and columnList, invalidate
add columns, rowspan, colspan, value, label